### PR TITLE
chore(align-deps): Recommend datetime-picker version compatible with new architecure & visionOS

### PR DESCRIPTION
**Pull request by Saadnajmi**
### Description

https://github.com/react-native-datetimepicker/datetimepicker/releases/tag/v8.0.0

This version supports the new architecture on 073+, and visionOS. Let's update our 0.73 profile preset

### Test plan

CI should pass.

### CODE REVIEW

File: .changeset/little-tips-jog.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rnx-kit/align-deps": patch
+---
+
+git push
```
Comment (tido64) [inline .changeset/little-tips-jog.md:5]: Can you write a better change log entry than this? 😆

File: packages/align-deps/src/presets/microsoft/react-native/profile-0.73.ts [hunk 2]
```diff
@@ -102,7 +102,7 @@ export const profile: Profile = {
   },
   "datetime-picker": {
     name: "@react-native-community/datetimepicker",
-    version: "^7.7.0",
+    version: "^8.0.0",
```
Comment (tido64) [inline packages/align-deps/src/presets/microsoft/react-native/profile-0.73.ts:105]: For profile 0.72 (and below), I think we need to change them to `>=7.7.0 <8.0.0`

image

File: .changeset/little-tips-jog.md (hunk 1 above)
Comment (Saadnajmi) [reply .changeset/little-tips-jog.md:5]: oof

**Review by tido64 — APPROVED** [review]
Thanks ❤️

**Pull request merged**